DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: featureselection
 Title: Feature Selection
-Version: 1.0.0.0
+Version: 1.1.0.100
```

Comment:
Why `100`?

Reply:
No reason in particular. Just odd to have a build number of zero.

Reply:
Shall I change it to `0`?

Reply:
Fixed